Internal Server Error: /2020/05/21/testpost/like
Traceback (most recent call last):
  File "e:\djangoproject\isp\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "e:\djangoproject\isp\lib\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "e:\djangoproject\isp\lib\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "e:\djangoproject\isp\lib\site-packages\django\contrib\auth\decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "E:\853506_Karpyza_Andrey\DjangoChat\views.py", line 124, in post_like
    'comment_form': comment_form,
UnboundLocalError: local variable 'comment_form' referenced before assignment
Internal Server Error: /admin/DjangoChat/profile/
Traceback (most recent call last):
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\backends\utils.py", line 86, in _execute
    return self.cursor.execute(sql, params)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\backends\sqlite3\base.py", line 396, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: UNIQUE constraint failed: DjangoChat_registrationmessage.profile_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "E:\DjangoProject\ISP\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "E:\DjangoProject\ISP\lib\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "E:\DjangoProject\ISP\lib\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\contrib\admin\options.py", line 607, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\utils\decorators.py", line 130, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\contrib\admin\sites.py", line 231, in inner
    return view(request, *args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\utils\decorators.py", line 43, in _wrapper
    return bound_method(*args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\utils\decorators.py", line 130, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\contrib\admin\options.py", line 1704, in changelist_view
    response = self.response_action(request, queryset=cl.get_queryset(request))
  File "E:\DjangoProject\ISP\lib\site-packages\django\contrib\admin\options.py", line 1390, in response_action
    response = func(self, request, queryset)
  File "E:\853506_Karpyza_Andrey\DjangoChat\admin.py", line 49, in send_confirmation
    msg = RegistrationMessage.objects.create(profile=profile, sended=False)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\models\query.py", line 433, in create
    obj.save(force_insert=True, using=self.db)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\models\base.py", line 745, in save
    self.save_base(using=using, force_insert=force_insert,
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\models\base.py", line 782, in save_base
    updated = self._save_table(
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\models\base.py", line 887, in _save_table
    results = self._do_insert(cls._base_manager, using, fields, returning_fields, raw)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\models\base.py", line 924, in _do_insert
    return manager._insert(
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\models\query.py", line 1204, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\models\sql\compiler.py", line 1391, in execute_sql
    cursor.execute(sql, params)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\backends\utils.py", line 100, in execute
    return super().execute(sql, params)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\backends\utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\backends\utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\backends\utils.py", line 86, in _execute
    return self.cursor.execute(sql, params)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\utils.py", line 90, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\backends\utils.py", line 86, in _execute
    return self.cursor.execute(sql, params)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\backends\sqlite3\base.py", line 396, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: UNIQUE constraint failed: DjangoChat_registrationmessage.profile_id
"POST /admin/DjangoChat/profile/ HTTP/1.1" 500 203403
Internal Server Error: /admin/DjangoChat/profile/
Traceback (most recent call last):
  File "E:\DjangoProject\ISP\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "E:\DjangoProject\ISP\lib\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "E:\DjangoProject\ISP\lib\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\contrib\admin\options.py", line 607, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\utils\decorators.py", line 130, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\contrib\admin\sites.py", line 231, in inner
    return view(request, *args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\utils\decorators.py", line 43, in _wrapper
    return bound_method(*args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\utils\decorators.py", line 130, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\contrib\admin\options.py", line 1704, in changelist_view
    response = self.response_action(request, queryset=cl.get_queryset(request))
  File "E:\DjangoProject\ISP\lib\site-packages\django\contrib\admin\options.py", line 1390, in response_action
    response = func(self, request, queryset)
  File "E:\853506_Karpyza_Andrey\DjangoChat\admin.py", line 49, in send_confirmation
    if not profile.verified and not RegistrationMessage.objects.get(profile=profile):
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\models\query.py", line 415, in get
    raise self.model.DoesNotExist(
DjangoChat.models.RegistrationMessage.DoesNotExist: RegistrationMessage matching query does not exist.
"POST /admin/DjangoChat/profile/ HTTP/1.1" 500 127533
Internal Server Error: /admin/DjangoChat/profile/
Traceback (most recent call last):
  File "E:\DjangoProject\ISP\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "E:\DjangoProject\ISP\lib\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "E:\DjangoProject\ISP\lib\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\contrib\admin\options.py", line 607, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\utils\decorators.py", line 130, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\contrib\admin\sites.py", line 231, in inner
    return view(request, *args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\utils\decorators.py", line 43, in _wrapper
    return bound_method(*args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\utils\decorators.py", line 130, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\contrib\admin\options.py", line 1704, in changelist_view
    response = self.response_action(request, queryset=cl.get_queryset(request))
  File "E:\DjangoProject\ISP\lib\site-packages\django\contrib\admin\options.py", line 1390, in response_action
    response = func(self, request, queryset)
  File "E:\853506_Karpyza_Andrey\DjangoChat\admin.py", line 50, in send_confirmation
    response = RegistrationMessage.objects.get(profile=profile)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\models\query.py", line 415, in get
    raise self.model.DoesNotExist(
DjangoChat.models.RegistrationMessage.DoesNotExist: RegistrationMessage matching query does not exist.
"POST /admin/DjangoChat/profile/ HTTP/1.1" 500 127525
Internal Server Error: /admin/DjangoChat/profile/
Traceback (most recent call last):
  File "E:\DjangoProject\ISP\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "E:\DjangoProject\ISP\lib\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "E:\DjangoProject\ISP\lib\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\contrib\admin\options.py", line 607, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\utils\decorators.py", line 130, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\contrib\admin\sites.py", line 231, in inner
    return view(request, *args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\utils\decorators.py", line 43, in _wrapper
    return bound_method(*args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\utils\decorators.py", line 130, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\contrib\admin\options.py", line 1704, in changelist_view
    response = self.response_action(request, queryset=cl.get_queryset(request))
  File "E:\DjangoProject\ISP\lib\site-packages\django\contrib\admin\options.py", line 1390, in response_action
    response = func(self, request, queryset)
  File "E:\853506_Karpyza_Andrey\DjangoChat\admin.py", line 51, in send_confirmation
    response = RegistrationMessage.objects.get(profile=profile)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "E:\DjangoProject\ISP\lib\site-packages\django\db\models\query.py", line 415, in get
    raise self.model.DoesNotExist(
DjangoChat.models.RegistrationMessage.DoesNotExist: RegistrationMessage matching query does not exist.
"POST /admin/DjangoChat/profile/ HTTP/1.1" 500 127508
